recipe/meta.yaml

```diff
@@ -10,18 +10,18 @@ source:
 
 build:
   noarch: python
-  number: 0
+  number: 1
```

Comment:
This is not really necessary with the patching. But not bad either.

Reply:
Bumping the build number is the right thing to do here